sdk/cosmosdb/cosmos/src/client/Container/ContainerRequest.ts

```diff
@@ -5,7 +5,15 @@ import { PartitionKeyDefinition } from "../../documents";
 import { VerboseOmit } from "../../utils/types";
 
 export interface ContainerRequest extends VerboseOmit<ContainerDefinition, "partitionKey"> {
-  /* Throughput for this container. */
+  /* Throughput for this container. Cannot use with maxThroughput */
   throughput?: number;
+  /* Max throughput for this container. Specify to use Autopilot */
```

Comment:
```suggestion
  /* Max throughput for this container. Specify to use Autoscale*/
```

FYI Offical name is "Autoscale"